eng/pipelines/smoke-tests.yml

```diff
@@ -11,34 +11,69 @@ jobs:
           OSVmImage: "ubuntu-16.04"
           TestTargetFramework: netcoreapp2.1
           CloudType: public
+          AdditionalParameters: $(publicAdditionalParameters)
+          Location: $(publicLocation)
         Windows_NetCoreApp (Public):
           OSName: "Windows"
           OSVmImage: "windows-2019"
           TestTargetFramework: netcoreapp2.1
           CloudType: public
+          AdditionalParameters: $(publicAdditionalParameters)
+          Location: $(publicLocation)
         Windows_NetFramework (Public):
           OSName: "Windows"
           OSVmImage: "windows-2019"
           TestTargetFramework: net461
           CloudType: public
+          AdditionalParameters: $(publicAdditionalParameters)
+          Location: $(publicLocation)
         MacOs (Public):
           OSName: "MacOS"
           OSVmImage: "macOS-10.15"
           TestTargetFramework: netcoreapp2.1
           CloudType: public
+          AdditionalParameters: $(publicAdditionalParameters)
+          Location: $(publicLocation)
         Windows_NetCoreApp (Gov):
           OSName: "Windows"
           OSVmImage: "windows-2019"
           TestTargetFramework: netcoreapp2.1
           CloudType: gov
+          AdditionalParameters: $(govAdditionalParameters)
+          Location: $(govLocation)
         Windows_NetFramework (Gov):
           OSName: "Windows"
           OSVmImage: "windows-2019"
           TestTargetFramework: net461
           CloudType: gov
+          AdditionalParameters: $(govAdditionalParameters)
+          Location: $(govLocation)
+        Windows_NetCoreApp (China):
+          OSName: "Windows"
+          OSVmImage: "windows-2019"
+          TestTargetFramework: netcoreapp2.1
+          CloudType: cn
```

Comment:
Is cn the general name used other places for the china cloud?

Reply:
It's the TLD for the authority host, however there's no consistency across these names. A better handle is the environment names themselves (e.g. `AzureCloud`, `AzureUSGovernment`, and `AzureChinaCloud`)